data/widgets/contentGroup/contentGroup.ui

```diff
@@ -29,6 +34,56 @@
             <property name="top_attach">0</property>
           </packing>
         </child>
+        <child>
+          <object class="GtkStack" id="log-button-stack">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="transition_duration">250</property>
+            <property name="transition_type">crossfade</property>
+            <child>
+              <object class="GtkButton" id="log-button">
+                <property name="label" translatable="yes">Download support info</property>
+                <property name="visible">True</property>
+                <property name="can_focus">True</property>
+                <property name="receives_default">True</property>
+                <property name="tooltip_text" translatable="yes">Click here to download a file with information that will help us track down the problem.</property>
+                <property name="halign">start</property>
+                <property name="valign">start</property>
+                <property name="image">image1</property>
+                <property name="image_position">right</property>
+                <property name="always_show_image">True</property>
+                <style>
+                  <class name="ContentGroupContentGroup__logButton"/>
+                </style>
+              </object>
+              <packing>
+                <property name="name">button</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="thank-you-label">
```

Comment:
Now that the log file gets created in a temporary directory, I think we must let the user to re-open the log as many times he wants. since I can easily imagine the user closing "gedit" by mistake, but then he won't be able to open the log again.

This would simplify patch as well.